_sass/_base.scss

```diff
@@ -11,6 +11,7 @@ $color-alpha--light: #f5f3ef;
 $color-black: #0a0a0a;
 $color-grey: #f8f9f9;
 $color-white: #fff;
+$color-blue: #323CFF;
```

Comment:
Why did you add a new color?

Reply:
I changed the style for links:

```css
a {
  color: $color-black;
  text-decoration: none;
  transition: color 0.4s ease-in-out;

  &:hover,
  &:active,
  &:focus {
    color: $color-blue;
  }
}
```

If you think the previous style was better, I can reverse this. Let me know what you think!

Reply:
The links are now on color-alpha by default and blue on hover, if you'd like me to change the stylr back to how it was before please add a new comment.